build/configs/artik055s/audio/defconfig

```diff
@@ -412,13 +412,6 @@ CONFIG_AUDIO_MAX_INPUT_CARD_NUM=2
 CONFIG_AUDIO_MAX_OUTPUT_CARD_NUM=2
 CONFIG_AUDIO_MAX_DEVICE_NUM=3
 # CONFIG_AUDIO_PROCESSING_FEATURES is not set
-
-#
-# Audio Processing Features
-#
-# CONFIG_AUDIO_SPEECH_DETECT_FEATURES is not set
-# CONFIG_AUDIO_KEYWORD_DETECT is not set
-# CONFIG_AUDIO_ENDPOINT_DETECT is not set
```

Comment:
Are these configs not used anymore..?

Reply:
I updated defconfig using `make menuconfig`. I don't care why this configs are removed.
Anyway these config are not set, so we don't consider them.